test/rpc-integration/resources/foundry-bug-report.tar.gz.kompile

```diff
@@ -0,0 +1,42 @@
+set -eux
+
+SCRIPT_DIR=$(dirname $0)
+PLUGIN_DIR=${PLUGIN_DIR:-""}
+NIX_LIBS=${NIX_LIBS:-""}
+
+
+if [ -z "$PLUGIN_DIR" ]; then
+    echo "PLUGIN_DIR required to link in a crypto plugin dependency"
+    exit 1
+else
+    for lib in libff libcryptopp libsecp256k1; do
+        LIBFILE=$(find ${PLUGIN_DIR} -name "${lib}.a" | head -1)
+        [ -z "$LIBFILE" ] && (echo "[Error] Unable to locate ${lib}.a"; exit 1)
+        PLUGIN_LIBS+="$LIBFILE "
+        PLUGIN_INCLUDE+="-I$(dirname $LIBFILE)/../include "
+    done
+    #PLUGIN_CPP=$(find ${PLUGIN_DIR}/plugin-c -name "*.cpp")
+    PLUGIN_CPP="${PLUGIN_DIR}/plugin-c/blake2.cpp ${PLUGIN_DIR}/plugin-c/crypto.cpp ${PLUGIN_DIR}/plugin-c/plugin_util.cpp"
+fi
+
+NAME=$(basename ${0%.tar.gz.kompile})
+NAMETGZ=$(basename ${0%.kompile})
+
+# unpack definition.kore and llvm_definition from tarball
+# into a directory named after the tarball
+cd $SCRIPT_DIR
+mkdir -p $NAME
+tar xzf ../test-$NAMETGZ -C $NAME definition.kore llvm_definition/
+
+# provide definition
+cp $NAME/definition.kore ${NAMETGZ}.kore
+
+# kompile llvm-definition to interpreter
+llvm-kompile $NAME/llvm_definition/definition.kore $NAME/llvm_definition/dt c -- \
+             -fPIC -std=c++17 -o interpreter \
+             $PLUGIN_LIBS $PLUGIN_INCLUDE $PLUGIN_CPP \
+             -lcrypto -lssl -lprocps $NIX_LIBS
```

Comment:
This script follows how we link plugin code to the llvm interpreter in `evm-semantics`:
- statically link in the plugin dependencies (`PLUGIN_LIBS`)
- compile (some of) the plugin's `*.cpp` files (`PLUGIN_CPP`), requiring the `include` directories of its dependencies (`PLUGIN_INCLUDE`)
- link (dynamically) to `crypto`, `ssl`, and `procps` (which is BTW Linux specific)
- access `NIX_LIBS` variable to obtain include and library directories for the latter

This has a couple of issues when run with `nix`:
1. the plugin dependencies are not actually built so the `LIBFILE` variable stays empty
2. there is no `NIX_LIBS` at the moment (would probably be easy enough to add?)